src/app/pages/e-commerce/e-commerce.component.html

```diff
@@ -0,0 +1 @@
+<p>e-commerce trabajando!</p>
```

Comment:
No me gusta este cambio por esta razón bal bla bla
```suggestion
<p>e-commerce works!</p>
```